SampleApps/ReceiverDemoSwift/DPLReceiverSwiftAppDelegate.swift

```diff
@@ -0,0 +1,31 @@
+import UIKit
+
+@UIApplicationMain
+class DPLReceiverSwiftAppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+    lazy var router: DPLDeepLinkRouter = DPLDeepLinkRouter()
+
+    func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
+        
+        // Route registration.
+        self.router["/product/:sku"] = DPLProductRouteHandler.self;
+        
+        self.router.registerHandlerClass(DPLMessageRouteHandler.self, forRoute: "/say/:title/:message")
```

Comment:
Maybe make it clear that you're showing registration using two different methods here?

```objc
// Register a class to a route using object subscripting
self.router["/product/:sku"] = DPLProductRouteHandler.self;

// Register a class to a route using the explicit registration call
self.router.registerHandlerClass(DPLMessageRouteHandler.self, forRoute: "/say/:title/:message")
```